R/font-bitstream-vera.R

```diff
@@ -34,21 +35,21 @@ font_bitstream_vera <- function(variant = "sans", style = "roman",
   ))
 }
 
-font_bitstream_vera_files <- list(
-  sans = c(
-    roman = "Vera",
-    bold = "VeraBd",
-    oblique = "VeraIt",
-    `bold-oblique` = "VeraBI"
+font_bitstream_vera_files <- font_files(font_bitstream_vera,
+  sans = list(
+    roman = font_sans("Vera"),
```

Comment:
ah yes but these are the fontconfig categories. The main data structure uses the fontconfig nomenclature since it is a superset of the R nomenclature. Then the R categories are recorded by the font constructors `font_sans(bold = FALSE, italic = FALSE)`, `font_serif(bold = FALSE, italic = FALSE)`, etc.

This way the regular getter provide access to all versions of a font, and you can use the new helpers provided in this PR (`font_info()`, `font_info_files()`) to request a font using the R categories.

Is this approach reasonable?